spec/Powertrain/Transmission.vspec

```diff
@@ -48,6 +48,7 @@
   max: 250
   unit: km/h
   description: Vehicle speed, as sensed by the gearbox.
+  deprecation: V2.1 removed because doubled with Vehicle.Speed
```

Comment:
We recently marked "Navigation.CurrentLocation.Speed" as deprecated with the rationale that it was a duplicate. Should then this datapoint also be marked as duplicate.  (datatype/limits not changed for datapoints that are depricated)